config/samples/example-test.sh

```diff
@@ -487,6 +487,21 @@ waitComponentPlanRetryTime "default" "do-once-nginx-sample-15.0.2" "5"
 info "5.7.4 verify this componentplan will failed, show error log"
 kubectl get cpl do-once-nginx-sample-15.0.2 --output='jsonpath={.status.conditions[?(@.type=="Actioned")]}'
 
+info "5.8 Verify that helm add repo with basic auth"
+kubectl apply -f config/samples/core_v1alpha1_repository_kubebb.yaml
+kubectl apply -f config/samples/core_v1alpha1_componentplan_chartmuseum.yaml
+waitComponentPlanDone "kubebb-system" "chartmuseum"
+export POD_NAME=$(kubectl get pods --namespace kubebb-system -l app.kubernetes.io/instance=chartmuseum -o jsonpath="{.items[0].metadata.name}")
+waitPodReady "kubebb-system" "app.kubernetes.io/instance=chartmuseum"
+kubectl port-forward $POD_NAME 8088:8080 --namespace kubebb-system &
+kubectl apply -f config/samples/core_v1alpha1_repository_chartmuseum.yaml
+curl -u admin:password http://localhost:8088/
+
+info "5.9 Verify that helm install with basic auth"
+curl -O https://charts.bitnami.com/bitnami/nginx-15.1.2.tgz
+curl -u admin:password --data-binary "@nginx-15.1.2.tgz" http://localhost:8088/api/charts
+kubectl apply -f config/samples/core_v1alpha1_componentplan_mynginx.yaml
```

Comment:
https://github.com/kubebb/core/blob/bb2eb94ee06e2e6b810ee839e85a5fff682362b1/config/samples/example-test.sh#L364-L368

like this, after apply componentplan, we should make sure it's actually running correctly.